core/domain/html_validation_service.py

```diff
@@ -238,7 +238,7 @@ def convert_to_textangular(html_data):
         elif tag.name == 'td' and tag.next_sibling:
             tag.insert_after(' ')
             tag.unwrap()
-        # Rows div and table both are replaced with p tag
+        # Divs and table rows both are replaced with p tag
```

Comment:
both are --> are both

(Also, if you don't mind, could you please correct the spelling of "apperance" in the line below? I know this was there before, but we might as well fix it. Thanks!)

Reply:
Done.